### PR TITLE
small utility to convert matches from .bin to .txt or .txt to .bin

### PR DESCRIPTION
**Pull request by yocabon**

### CODE REVIEW

File: src/software/SfM/main_ConvertMatches.cpp [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+// Copyright (c) 2012, 2013, 2015 Pierre MOULON.
+
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "openMVG/matching/indMatch.hpp"
+#include "openMVG/matching/indMatch_utils.hpp"
+#include "openMVG/matching/svg_matches.hpp"
```
Comment (pmoulon) [inline src/software/SfM/main_ConvertMatches.cpp:11]: not needed

File: src/software/SfM/main_ConvertMatches.cpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+#include "openMVG/image/image_io.hpp"
```
Comment (pmoulon) [inline src/software/SfM/main_ConvertMatches.cpp:12]: not needed

**Review by pmoulon — CHANGES_REQUESTED** [review]
Thank you for your contribution
I just advised some cleanup and directly using Load/Save function from the feature library files.
It will make the code even shorter and simpler.

**Review by yocabon — COMMENTED** [review]
Done

**Comment by pmoulon** [comment]
LGTM, thank you for your contribution

**Pull request merged**